src/taxonomy/data/apiHooks.test.jsx

```diff
@@ -75,3 +78,38 @@ describe('useDeleteTaxonomy', () => {
     });
   });
 });
+
+describe('useTaxonomyDetailDataStatus', () => {
+  it('should return status values', () => {
+    const status = {
+      isError: false,
+      error: undefined,
+      isFetched: true,
+      isSuccess: true,
+    };
+
+    useQuery.mockReturnValueOnce(status);
+
+    const result = useTaxonomyDetailDataStatus(0);
+
+    expect(result).toEqual(status);
+  });
+});
+
+describe('useTaxonomyDetailDataResponse', () => {
+  it('should return data when status is success', () => {
+    useQuery.mockReturnValueOnce({ isSuccess: true, data: 'data' });
+
+    const result = useTaxonomyDetailDataResponse();
+
+    expect(result).toEqual('data');
+  });
+
+  it('should return undefined when status is not success', () => {
+    useQuery.mockReturnValueOnce({ isSuccess: false });
+
+    const result = useTaxonomyDetailDataResponse();
+
+    expect(result).toBeUndefined();
+  });
+});
```

Comment:
These tests are not testing anything, please remove them in favour of testing the component directly.

Reply:
Done: aa1031e2fe7504081c4fd8cf20f4ebfcbfb17382